<?xml version="1.0" ?>
<launch>
    <!-- Robot Description -->
    <param name="robot_description" textfile="$(find omni_wheel_robot)/urdf/robot_base.urdf" />

    <!-- Joint State Publisher -->
    <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
    <!-- Robot State Publisher -->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

    <!-- Holonomic Controller -->
    <!-- <node name="holonomic_controller" pkg="omni_wheel_robot" type="holonomic_controller.py" output="screen" /> -->

    <!-- Odometry Publisher -->
    <node name="odom_publisher" pkg="omni_wheel_robot" type="odom_publisher.py" output="screen" />

    <!-- Static Transform Publisher for Laser (if applicable) -->
    <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" args="0.275 0 0 0 0 0 base_link laser 1000" />

    <!-- Static Transform Publisher for Map to Odom -->
    <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 0 0 0 map odom 50" />

</launch>
